client/src/components/ListDeleter.js

```diff
@@ -2,13 +2,20 @@ import React from "react";
 import PropTypes from "prop-types";
 import styled from "@emotion/styled";
 
-function ListDeleter({ onCancel }) {
+function ListDeleter({ onCancel, onDeleteConfirm, id }) {
   return (
     <Wrapper>
       <h4>Do you want to delete this list?</h4>
       <div>
         <button onClick={() => onCancel(false)}>Cancel</button>
-        <button onClick={() => onCancel(false)}>Proceed</button>
+        <button
+          onClick={() => {
+            onDeleteConfirm(id);
+            onCancel(false);
+          }}
```

Comment:
Why is it necessary to call `onCancel`?
Shouldn't it enough to just call the confirmation?

Reply:
onCancel will close the overlay window